resources/css/thrivedesk.css

```
@@ -11,162 +14,1554 @@ eg: #thrivedesk .search{ display: flex !important; }
+		padding: 0.375rem 0.625rem 0.375rem 0.5rem;
+		border-radius: calc(var(--radius) - 2px);
+		margin-left: -0.5rem;
+	}
+	.td-back-link:hover { color: hsl(var(--foreground)); background: hsl(var(--accent)); }
+	.td-back-link svg { width: 1rem; height: 1rem; }
+
+	/* header block: meta row + subject. Sticky on desktop, sits under
+	   the toolbar as the page scrolls. */
+	.td-conv-header {
+		margin-bottom: 1rem;
+		/*position: sticky;*/
```

Stylelint (17.13.0)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)